### PR TITLE
Serialize full txns

### PR DESCRIPTION
**Pull request by redxaxder**

### CODE REVIEW

File: shelley/chain-and-ledger/executable-spec/test/Tests.hs [hunk 1]
```diff
@@ -3,14 +3,16 @@ import           Test.Tasty
 import           PropertyTests (minimalPropertyTests)
 import           STSTests (stsTests)
 import           Test.Serialization (serializationTests)
+import           Test.CDDL (cddlTests)
 import           UnitTests (unitTests)
 
 tests :: TestTree
 tests = testGroup "Ledger with Delegation"
-  [ unitTests
+  [ cddlTests
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/executable-spec/test/Tests.hs]: :tada:

**Comment by mgudemann** [comment]
I just tried this and get ```Exception: cddl: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)```

**Comment by mgudemann** [comment]
with `cddl` installed, I get ```Exception: diag2cbor.rb: createProcess: runInteractiveProcess: exec: does not exist```

**Comment by JaredCorduan** [comment]
> with `cddl` installed, I get `Exception: diag2cbor.rb: createProcess: runInteractiveProcess: exec: does not exist`

you need to `gem install cbor-diag` or run the tests with nix.

**Comment by JaredCorduan** [comment]
The only thing missing from this PR is proper CI support. I think it is a good idea to comment out the cddl tests for now and merge this PR. (and then work on getting the needed cbor tools working in bulidkite.)

**Pull request merged**